doc-Appliance_Hardening_Guide/cfme/master.adoc

```diff
@@ -0,0 +1,30 @@
+= Appliance Hardening Guide
+:vernum: 4.1
+:doctype: book
+:toc: left
+:toclevels: 3
+:numbered!:
+:experimental:
```

Comment:
The rest of these settings are not currently used by AsciiBinder, as the documents are cast into the page template which is located at [_templates/page.html.erb](https://github.com/ManageIQ/manageiq_docs/blob/master/_templates/page.html.erb) inside this docs repo.